Stylesheets/components/badge.css

```diff
@@ -0,0 +1,73 @@
+@import url(/Stylesheets/main.css);
+
+.bedge {
+  margin: 2rem;
+}
+.bg {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  color: white;
+  height: 2rem;
+  width: 2rem;
+  background-color: red;
+  border-radius: 50%;
+  position: relative;
+}
+
+.ml {
+  left: 4rem;
+  bottom: 6rem;
+}
+
+.cart {
+  left: 12.5rem;
+  bottom: 8rem;
+}
+.phone {
+  left: 19.7rem;
+  bottom: 10rem;
+}
+.insta {
+  left: 27.3rem;
+  bottom: 12rem;
+}
+
+.hero-img {
+  height: 100px;
+  width: auto;
+  border-radius: 50%;
+  margin: 2rem;
+}
+
+.social {
+  margin: 2rem;
+  height: 30px;
```

Comment:
Use either rem or px , don't switch units in between

Reply:
ok i have taken rem everywhere